src/main/java/org/broadinstitute/hellbender/tools/exome/BalancedHeterozygousPileupPriorModel.java

```diff
@@ -12,17 +12,15 @@
  *
  * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
  */
-public class BalancedHeterozygousPileupPriorModel extends HeterozygousPileupPriorModel {
+public final class BalancedHeterozygousPileupPriorModel extends HeterozygousPileupPriorModel {
 
     /**
-     * Calculates the log likelihood of heterzygosity assuming allele fraction = 1/2
+     * Calculates the log likelihood of heterozygosity assuming allele fraction = 1/2
      * @param coeffs list of (alpha, beta) tuples
-     * @param minErrorProbability the theoretical minimum error probability for each nucleotide in the pileup
-     * @return
+     * @return any double value
      */
-    public double getHetLogLikelihood(final List<ImmutablePair<Double, Double>> coeffs,
-                                      final double minErrorProbability) {
-        return getHetLogLikelihoodFixedAlleleFraction(0.5, coeffs, minErrorProbability);
+    @Override
+    public double getHetLogLikelihood(final List<ImmutablePair<Double, Double>> coeffs) {
```

Comment:
Change the signature of these class and its parent to point to the less stringent class possible. I think that ```ImmutablePair``` extends ```Pair``` (there is mutable sibling class). 

So do: ```List<? extends Pair<Double, Double>> coeffs```.

If in turn, the delegated to method's code can handle Collections or event Iterables perhaps you want to also change List to that:

so ```Collection<? extends Pair<Double, Double>>```
or ```Iterable<? extends Pair<Double, Double>>```

You can go even further and don't assume they are doubles but that they can be any number type:

```Iterable<? extends Pair<? extends Number, ? extends Number>>```

But perhaps in that case you will loose the auto-unboxing ... you would need to get the double value explictly example: ```pair.getLeft().doubleValue()```.